app/components/Session/NewSessionForm.js

```diff
@@ -81,48 +82,51 @@ export default class NewSessionForm extends Component {
     const {setCapabilityParam, caps, addCapability, removeCapability, saveSession, hideSaveAsModal, saveAsText, showSaveAsModal, setSaveAsText} = this.props;
 
     return <div>
-      <h4>Desired Capabilities</h4>
-      <Row> 
-      <Col span={12}>
+      <Row type="flex" align="top" justify="start" className={SessionStyles.capsFormRow}>
+      <Col order={1} className={SessionStyles.capsFormCol}>
         <Form inline>
           {caps.map((cap, index) => {
             return <div key={index} className={SessionStyles['desired-capabilities-form-container']}>
               <FormItem>
                 <Input placeholder='Name' value={cap.name} onChange={(e) => setCapabilityParam(index, 'name', e.target.value)}/>
               </FormItem>
               <FormItem>
-                <select onChange={(e) => this.handleSetType(index, e.target.value)} value={cap.type}>
-                  <option value='text'>text</option>
-                  <option value='boolean'>boolean</option>
-                  <option value='number'>number</option>
-                  <option value='json_object'>JSON object</option>
-                  <option value='file'>filepath</option>
-                </select>
+                <Select onChange={(val) => this.handleSetType(index, val)} defaultValue={cap.type} size="large" style={{width: 120}}>
```

Comment:
when I tried `value`, I found that I couldn't actually change the value in the drop-down. *shrug*